### DISCUSSION TITLE
12 hour delete

### DISCUSSION THREAD

**Discussion by adamweld** [joeycastillo/Sensor-Watch#222] [General]
Has anyone played with completely deleting 12 hour mode from the firmware? 

To be honest, this project is cool as hell but the real reason I bought a board was to do exactly this ^, I got really tired of my wrist knocking the watch into 12h mode. I understand this is a lot more difficult with the current firmware, but, I'd sleep easier knowing the mode had been completely eradicated.

If not, no worries, I'll take a crack at it!

**Comment by shmoun** [comment]
A while ago I stumbled upon this commit: https://github.com/joeycastillo/Sensor-Watch/commit/f84391e83a99fb34beb2b6566df2e05fe6230099 

I can't find the original blog post that linked to it, so this is the only reference i have left :/

I was trying to do the exact same thing. This commit removes both 12h-  and imperial support entirely.

It worked in the emulator, before i received the board, but since watch_faces of the future might still expect support to be present I settled for setting the clock to 24h by default and keep support for 12h and imperial in my own branches.

**Reply by bademux** [reply]
LCD 24h our symbol feels bit unnatural. I think it should be repurposed to something more useful.
